packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/MultiWorkspaceDropdown/internal/MultiWorkspaceDropdownWorkspacesListComponents.tsx

```diff
@@ -52,37 +45,10 @@ export const MultiWorkspaceDropdownWorkspacesListComponents = () => {
         }}
       />
       <DropdownMenuSeparator />
-      <DropdownMenuItemsContainer>
-        {workspaces
-          .filter(
-            (workspace) =>
-              workspace.id !== currentWorkspace?.id &&
-              workspace.displayName
-                ?.toLowerCase()
-                .includes(searchValue.toLowerCase()),
-          )
-          .map((workspace) => (
-            <UndecoratedLink
-              key={workspace.id}
-              to={buildWorkspaceUrl(getWorkspaceUrl(workspace.workspaceUrls))}
-              onClick={(event) => {
-                event?.preventDefault();
-                handleChange(workspace);
-              }}
-            >
-              <MenuItemSelectAvatar
-                text={workspace.displayName ?? '(No name)'}
-                avatar={
-                  <Avatar
-                    placeholder={workspace.displayName || ''}
-                    avatarUrl={workspace.logo ?? DEFAULT_WORKSPACE_LOGO}
-                  />
-                }
-                selected={currentWorkspace?.id === workspace.id}
-              />
-            </UndecoratedLink>
-          ))}
-      </DropdownMenuItemsContainer>
+      <MemberWorkspaces searchValue={searchValue} />
```

Comment:
As discussed the conditions in MultiWorkspaceDropdownDefaultComponents.tsx should be reviewed to display invitations even if I have less than 3 workspace (to test it we had to change both conditions value workspaces.length > 1 and workspaces.length > 4 to 0)